snooty/types.py

```diff
@@ -36,6 +36,7 @@
 FileSource = Union[Path, str]
 PAT_VARIABLE = re.compile(r"{\+([\w-]+)\+}")
 PAT_GIT_MARKER = re.compile(r"^<<<<<<< .*?^=======\n.*?^>>>>>>>", re.M | re.S)
+IMAGE_SIZING_EXT = [".png", ".avif", ".jpg", ".jpeg", ".webp"]
```

Comment:
Since this is only used for member existence, it should be a should be a set `{...}`

Reply:
⚡ ~1/4 time for the win. updated!